sdk/python/README.md

```diff
@@ -59,36 +65,15 @@ from pprint import pprint
 
 ## Documentation for API Endpoints
 
-All URIs are relative to *http://localhost*
```

Comment:
@ahg-g is there a case to keep the jobset models?

I'm not sure what value these docs provide but we do have the documentation for jobset.

Reply:
I don't think so, but I wouldn't wrestle with it if automatically generated